### ISSUE TITLE
JAWS 2018 reads "x" when using MathJax in <label> with radio button

### ISSUE THREAD

**Issue by lud2k** [FreedomScientific/VFO-standards-support#25]
JAWS 2018 on IE11 does not read MathJax content that is in a \<label\> tag next to a radio button.
Instead it reads "x", "x radio button".

I have tried to find a solution to this issue but couldn't. Having "Supportive MathML" enabled does not change anything. Still reads "x radio button".

This can be reproduce with the following code with version 2.7.2 of MathJax.

```
<html>
    <head>
        <title>MathJax TeX Test Page</title>
        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <script type="text/x-mathjax-config">
            MathJax.Hub.Config({
                tex2jax: {inlineMath: [["$","$"],["\\(","\\)"]]}
            });
        </script>
        <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/MathJax.js?config=TeX-AMS_HTML-full"></script>
    </head>
    <body>
        <form>
            <p>
                What is the correct answer?
            </p>
            <label>
                <input type="radio" name="answer" value="0" />
                <div>
                    \( 0.48 \)
                </div>
            </label>
            <label>
                <input type="radio" name="answer" value="1" />
                <div>
                    \( 1.16 \)
                </div>
            </label>
            <label>
                <input type="radio" name="answer" value="2" />
                <div>
                    \( 12.92 \)
                </div>
            </label>
            <label>
                <input type="radio" name="answer" value="3" />
                <div>
                    \( 13.53 \)
                </div>
            </label>
        </form>
    </body>
</html>
```

Any ideas what could be causing this?

I have also opened an issue on the MathJax side:
https://github.com/mathjax/MathJax/issues/1874

Thanks,
Ludo.

**Comment by stevefaulkner** [comment]
Confirm behaviour JAWS 18.0.453.0 +IE11. 
Modified and reduced test case: https://codepen.io/stevef/pen/ZaqWzB
in the test case JAWS will announce the label, but prefixes it with 'x'. don't know where the x is coming from, needs further testing.

**Comment by stevefaulkner** [comment]
JAWS Bug 97955 filed - reads "x" when using MathJax in <label> with radio button
thanks for reporting @lud2k

**Comment by frex65** [comment]
Have tested using JAWS version 2020.2004.06 and can confirm this is fixed.
